apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -23,9 +31,8 @@ const CurriculumCatalogCard = ({
   imageSrc = 'https://images.code.org/0a24eb3b51bd86e054362f0760c6e64e-image-1681413990565.png',
   subjects = [],
   topics = [],
-  isTranslated = false,
-  isEnglish,
   pathToCourse,
+  ...props
```

Comment:
Rather than explicitly passing through all these props, I opted for fewer lines –– push back if you think this is less clear

Reply:
I'm okay with it, but just out of curiosity why did you also lump in `isTranslated` and `isEnglish` into it and not others?

Reply:
Good question –– these are the only two props that (a) didn't need a default (`isTranslated` being undefined will be consistent with it being false), and (b) wasn't used in a computation before passing it down.

I'm wondering if it'd make sense to remove the CustomizableCatalogCard component all together and just have one component –– but that'd be a future refactor. If you have a strong opinion either way, please let me know!